kratos/utilities/geometry_utilities.h

```diff
@@ -55,6 +55,114 @@ class KRATOS_API(KRATOS_CORE) GeometryUtils
      */
     static std::string GetGeometryName(const GeometryData::KratosGeometryType TypeOfGeometry);
 
+    /**
+     * @brief Returns the local coordinates of a given arbitrary point for a given linear tetrahedra
+     * @details Based on https://www.colorado.edu/engineering/CAS/courses.d/AFEM.d/AFEM.Ch09.d/AFEM.Ch09.pdf. Section 9.1.6
+     * @param rGeometry The geometry to be considered
+     * @param rResult The vector containing the local coordinates of the point
+     * @param rPoint The point in global coordinates
+     * @return The vector containing the local coordinates of the point
+     */
+    template<class TGeometryType>
+    static inline typename TGeometryType::CoordinatesArrayType& PointLocalCoordinatesTetrahedra3D4N(
```

Comment:
As far as I understand, you aim to use the simplex formula for any tetrahedron featuring planar faces. If so, I think the name of the method is misleading as you make explicit `Tetrahedra3D4N` while the argument accepts any tetrahedron.

Having said this, I'd suggest something like `PointLocalCoordinatesPlanarFaceTetrahedra` or similar.

Reply:
Precisely I changed the name specifying that is just for the simplex, when using in higher order tetrahedra it will work only when the higher order element is geometrically a simplex

Reply:
Fair enough. Indeed, usages of this must be through the geometry API so the naming shouldn't be a problem at all. In any case, thanks for taking my suggestion into consideration.